CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## v3.2.2
+* Fixed random failures during starting operation caused by wrong CSRF token handling
+* Files with same digest but different names will not be reuploaded
+* Fixed filtration by namespace in cf mta command
+* Improved resilience during deploy from URL
+* Fixed enablement of skip-ssl-validation
+* Binaries are now statically linked
```

Comment:
Can we explain what was the issuue?
E.g link to https://github.com/cloudfoundry/multiapps-cli-plugin/issues/186 or so

Reply:
Done